package.json

```diff
@@ -2192,15 +2192,13 @@
         "bootstrap": "^4.3.1",
         "bootstrap-less": "^3.3.8",
         "cross-fetch": "^3.1.5",
-        "crypto": "^1.0.1",
```

Comment:
No idea why this was added in the first place, its an empty package
Possible some hacky way to dismiss some warnings , no idea.
Removing this as this is not required and marked as deprecated